demo_flake8_bugbear.py

```diff
@@ -1,30 +1,29 @@
 import socket
 
 
-23 < 42
-
-
 def printer_is_available(printer_host: str, printer_port: int) -> bool:
     """Check if printer is available."""
     try:
         with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as sock:
             sock.settimeout(0.5)
             sock.connect((printer_host, printer_port))
             return True
-    except:
+    except BaseException:
```

Comment:
Do not catch BaseException unless you have a solid reason for this. See details: [stackoverflow](https://stackoverflow.com/questions/7160983/catching-all-exceptions-in-python#:~:text=The%20difference%20between%20catching%20Exception,they%20inherit%20directly%20from%20BaseException%20.)